### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by RobLoach**
:warning: See questions below.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,17 +5,10 @@ var fs   = require('fs');
...
-  return jade.compileClientWithDependenciesTracked(source, options);
-};
-exports.compileFile = function (path, options) {
-  var fn = jade.compileFile(path, options);
-  return {fn: fn, dependencies: fn.dependencies}
-};
+
+exports.compile = jade.compile;
+exports.compileClient = jade.compileClientWithDependenciesTracked;
+exports.compileFile = jade.compileFile;
```
Comment (RobLoach) [inline index.js:11]: return `{fn: fn, dependencies: fn.dependencies}`?

Isn't this something `jstransformer` should handle?

Comment (TimothyGu) [inline index.js:11]: Yes, I believe so. Have you tested it?

Comment (RobLoach) [inline index.js:11]: I've switched it to just `exports.compile = jade.compile` in assumption that jstransformer will handle it. Looks like it's passing.

**Comment by TimothyGu** [comment]
LGTM.

File: index.js (hunk 1 above)
Comment (ForbesLindesay) [inline index.js:11]: No. Jstransformer handles the case of just returning a function when dependencies are not known (or can't be tracked for some reason).  The format of returning a function that has a dependencies property is Jade specific.

File: package.json [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
   },
   "scripts": {
     "test": "node test",
-    "coverage": "istanbul cover test",
-    "coveralls": "npm run coverage && cat ./coverage/lcov.info | coveralls"
+    "coverage": "istanbul cover test"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/jstransformers/jstransformer-jade.git"
```
Comment (ForbesLindesay) [inline package.json:23]: This should still be Jade, not foo

Comment (RobLoach) [inline package.json:23]: :+1:

File: index.js (hunk 1 above)
Comment (TimothyGu) [inline index.js:11]: @ForbesLindesay Okay got it.

**Pull request closed without merging**